src/Elders.Cronus/Discoveries/AssemblyLoader.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+using System.Threading;
+using Elders.Cronus.Logging;
+
+namespace Elders.Cronus.Discoveries
+{
+    internal class AssemblyLoader
+    {
+        static readonly ILog log = LogProvider.GetLogger(nameof(AssemblyLoader));
+
+        static int shouldLoadAssembliesFromDir = 1;
+        static string[] excludedAssemblies = new string[] { "apphost.exe", "clrcompression.dll", "clretwrc.dll", "clrjit.dll", "coreclr.dll", "dbgshim.dll", "hostfxr.dll", "hostpolicy.dll", "mscordaccore.dll", "mscordaccore_amd64_amd64_4.6.26814.03.dll", "mscordbi.dll", "mscorrc.debug.dll", "mscorrc.dll", "sos.dll", "sos_amd64_amd64_4.6.26814.03.dll", "ucrtbase.dll" };
```

Comment:
What if these assemblies change or there is a new version for mscordaccore_amd64_amd64_x.x.xxxxx.xx?

Reply:
It will write a warning in the logs